Transformations/DifferenceGraph-to-CityGML3/DifferenceGraph2RDF.py

```diff
@@ -88,59 +98,106 @@ def main():
 
 
     # add VersionTransition and Version
-    source_version_uri = URIRef( DATA.version_ + args.time_stamps[0] )
-    target_version_uri = URIRef( DATA.version_ + args.time_stamps[1] )
-    versiontransition_uri = URIRef( DATA.versiontransition_ + f'{args.time_stamps[0]}_{args.time_stamps[1]}' )
+    source_version_uri = URIRef( DATA.version_ + args.version_ids[0] )
+    target_version_uri = URIRef( DATA.version_ + args.version_ids[1] )
+    versionTransition_uri = URIRef( DATA.versionTransition_ + f'{args.version_ids[0]}_{args.version_ids[1]}' )
 
     output_graph.add( (source_version_uri, RDF.type, VERS.Version) )
     output_graph.add( (source_version_uri, RDF.type, OWL.NamedIndividual) )
     output_graph.add( (target_version_uri, RDF.type, VERS.Version) )
     output_graph.add( (target_version_uri, RDF.type, OWL.NamedIndividual) )
-    output_graph.add( (versiontransition_uri, RDF.type, VERS.VersionTransition) )
-    output_graph.add( (versiontransition_uri, RDF.type, OWL.NamedIndividual) )
+    output_graph.add( (versionTransition_uri, RDF.type, VERS.VersionTransition) )
+    output_graph.add( (versionTransition_uri, RDF.type, OWL.NamedIndividual) )
     transaction_member_uri = URIRef( VERS.VersionTransition + '.transaction' )
     from_uri = URIRef( VERS.VersionTransition + '.from' )
     to_uri = URIRef( VERS.VersionTransition + '.to' )
-    output_graph.add( (versiontransition_uri, from_uri, source_version_uri) )
-    output_graph.add( (versiontransition_uri, to_uri, target_version_uri) )
+    output_graph.add( (versionTransition_uri, from_uri, source_version_uri) )
+    output_graph.add( (versionTransition_uri, to_uri, target_version_uri) )
+
+    # add version and versionTransition timestamps if provided
+    if args.existence_time_stamps is not None:
```

Comment:
Add detailed comments to explain the if conditions.

Reply:
added explination in the [argparse documentation](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/Transformations/DifferenceGraph-to-CityGML3/DifferenceGraph2RDF.py#L19) for each type of timestamp